packages/SwingSet/src/kernel/kernelSyscall.js

```diff
@@ -88,63 +86,53 @@ export function makeKernelSyscallHandler(tools) {
   /**
    * Execute one step of iteration over a range of vatstore keys.
    *
-   * @param {string} vatID  The vat whose vatstore is being iterated
-   * @param {string} priorKey  The key that was returned by the prior cycle of
+   * @param {string} vatID The vat whose vatstore is being iterated
+   * @param {string} priorKey The key that was returned by the prior cycle of
    *   the iteration, or '' if this is the first cycle
-   * @param {string} lowerBound  The lower bound of the iteration range
-   * @param {string} [upperBound]  The uppper bound of the iteration range.  If
+   * @param {string} lowerBound The lower bound of the iteration range
+   * @param {string} [upperBound] The uppper bound of the iteration range. If
    *   omitted, this defaults to a string equivalent to the `lowerBound` string
    *   with its rightmost character replaced by the lexically next character.
    *   For example, if `lowerBound` is 'hello', then `upperBound` would default
    *   to 'hellp')
-   *
-   * @returns {['ok', [string, string]|[undefined, undefined]]} A pair of a
-   *   status code and a result value.  In the case of this operation, the
-   *   status code is always 'ok'.  The result value is a pair of the key and
-   *   value of the first vatstore entry whose key is lexically greater than
-   *   both `priorKey` and `lowerBound`.  If there are no such entries or if the
-   *   first such entry has a key that is lexically greater than or equal to
+   * @returns {['ok', [string, string] | [undefined, undefined]]} A pair of a
+   *   status code and a result value. In the case of this operation, the status
+   *   code is always 'ok'. The result value is a pair of the key and value of
+   *   the first vatstore entry whose key is lexically greater than both
+   *   `priorKey` and `lowerBound`. If there are no such entries or if the first
+   *   such entry has a key that is lexically greater than or equal to
    *   `upperBound`, then result value is a pair of undefineds instead,
    *   signalling the end of iteration.
    *
-   * Usage notes:
+   *   Usage notes:
```

Comment:
Interesting why it's adding indentation here. I believe the intent was to make this a note for the function definition, but the result of putting it after `@returns` would make it included in the return value description ?

Reply:
That's right. IDE sees all this text as description of `@returns`. I think anything that's about the function as a whole has to go above the tags.

Another example of the formatter revealing to us how the JSDoc will be parsed.

Reply:
Presumably the IDE would have shown the original text as associated with the `@returns`, so this is another case of helping the programmer help the IDE?